Manifest.toml

```diff
@@ -0,0 +1,239 @@
+[[Base64]]
```

Comment:
Please don't include a Manifest file

Reply:
Sorry, I'm not sure how I managed to inadvertently add that.